Neos.Fusion/Classes/Core/Cache/ParserCache.php

```diff
@@ -55,9 +55,12 @@ public function cacheForFusionFile(?string $contextPathAndFilename, \Closure $ge
         if ($contextPathAndFilename === null) {
             return $generateValueToCache();
         }
-        if (str_contains($contextPathAndFilename, '://')) {
+        if (str_contains($contextPathAndFilename, 'resource://')) {
             $contextPathAndFilename = $this->getAbsolutePathForPackageRessourceUri($contextPathAndFilename);
         }
+        if (str_contains($contextPathAndFilename, 'nodetypes://')) {
+            $contextPathAndFilename = $this->getAbsolutePathForNodetypesUri($contextPathAndFilename);
```

Comment:
Nitpick: "..NodeTypes" is camel-cased in other places IIRC
```suggestion
            $contextPathAndFilename = $this->getAbsolutePathForNodeTypesUri($contextPathAndFilename);
```